### PR TITLE
Add config option to send all log lines to stdout

### PR DESCRIPTION
**Pull request by analogue**
This is a useful option when diagnosing failures in docker containers. Being able to send all scribe output to stdout where it can easily be inspected via `docker logs` is so much easier and straightforward than figuring out whether logs are being written to the filesystem or scribe.  This is especially useful when things fail.

DONE: I'll update docs and unit tests if there is no general opposition to this feature.

### CODE REVIEW

**Comment by analogue** [comment]
@ecanzonieri @yyejun what say you?

**Comment by yyejun** [comment]
lg2m

**Comment by ecanzonieri** [comment]
This option sounds good to me. Let's fix docs and add tests.

**Comment by analogue** [comment]
Ok, docs and unit tests added. Also moved release notes to root dir and symlinked to ```docs/source```

File: tests/test_clog.py [hunk 1]
```diff
@@ -135,3 +136,19 @@ def test_scribify(self):
         T.assert_equal(scribify("this is a test"), "this_is_a_test")
         T.assert_equal(scribify("this\0is a-test\n\n"), "this_is_a-test__")
         T.assert_equal(scribify(u'int\xe9rna\xe7ionalization'), 'int_rna_ionalization')
+
+
+class TestStdoutLogger(object):
+
+    def test_log_line_and_close(self):
+        with mock.patch('sys.stdout') as mock_stderr:
```
Comment (ecanzonieri) [inline tests/test_clog.py]: rename mock_stderr to mock_stdout?

Comment (analogue) [inline tests/test_clog.py]: good catch

**Comment by ecanzonieri** [comment]
+1

**Pull request merged**